### ISSUE TITLE
ERROR: local variable 'ASPERA_PRIVATE_KEY' referenced before assignment

### ISSUE THREAD

**Issue by tseemann** [enasequence/enaBrowserTools#21]
Not sure if i'm doing something silly?

```
% cat aspera.ini

[aspera]
ASPERA_BIN = ascp
ASPERA_PRIVATE_KEY = /home/linuxbrew/.linuxbrew/etc/asperaweb_id_dsa.openssh
ASPERA_OPTIONS = -QT
ASPERA_SPEED = 300M

% ./enaDataGet -a aspera.ini -f fastq SRR3309615

ERROR: cannot read aspera settings from aspera.ini
local variable 'ASPERA_PRIVATE_KEY' referenced before assignment
```

**Comment by tseemann** [comment]
I notice also the code refers to `ASPERA_LICENSE` and `os.path.exists(ASPERA_LICENSE)`. 

I don't have this file, I use this `ascp` installation on the command line:
http://download.asperasoft.com/download/sw/connect/3.6.2/aspera-connect-3.6.2.117442-linux-64.tar.gz

**Comment by nicsilvester** [comment]
I'll have a look into this. I thought I had removed the aspera licence reference so that is one fix I know I need to make. For the file, which python version are you using? My tests work but I may not have fully covered both versions in my test cases.

**Comment by nicsilvester** [comment]
fixed in v1.3.2
Sorry, this was a problem with the python3 version only
